job_templates/service_action_-_ansible_default.erb

```diff
@@ -12,7 +12,7 @@ template_inputs:
     At least one of state and enabled are required. Note that reloaded will start
     the service if it is not already started, even if your chosen init system wouldn't
     normally.
-  options: "started\r\nstopped\r\nrestarted\r\nreloaded"
+  options: "started\r\nstopped\r\nrestarted\r\nreloaded\nenable\ndisable"
```

Comment:
Setting `state` to `enable` would lead to passing `enable` into ansible's `service.state` which it wouldn't understand. These options have to directly map to choices for `state`, see [ansible docs](https://docs.ansible.com/ansible/latest/modules/service_module.html)

Reply:
i see the `enabled` flag at the documentation.
How can I set it using the `options` to be true or false?
Or should I use something else.

Reply:
There's already an input for it here https://github.com/theforeman/community-templates/pull/480/files#diff-28abe89ce1acc27089ab7963b059131cR36

Reply:
thank you